interfaces/IBrowser.h

```diff
@@ -90,8 +98,12 @@ namespace Exchange {
             virtual void VisibilityChange(const bool hidden) = 0;
             // @brief Notifies that the web page requests to close its window
             virtual void PageClosure() = 0;
-            /* @json:omit */
+            // @brief A Base64 encoded JSON message from legacy $badger bridge
+            // @param message BridgeQuery string
             virtual void BridgeQuery(const string& message) = 0;
+            // @brief Signals a state change of the Browser
```

Comment:
I guess this is from the IStateControl interface. Is it not possible to get it from that interface, see: https://github.com/rdkcentral/Thunder/blob/master/Source/plugins/IStateControl.h#L49